src/components/faceVerification/screens/VerificationScreen.js

```diff
@@ -155,6 +155,7 @@ const FaceVerification = ({ screenProps }) => {
   const sdkExceptionHandler = useCallback(
     exception => {
       fireEvent(FV_ZOOMFAILED)
+      trackAttempt(exception)
```

Comment:
@roma-hladilka 
only user-related errors should be tracked as 'attempt'
SDK exception are critical mostly, so they shows 'unexpected' error immediately